### PR TITLE
Changelog update for 2.4 branch 

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by ronso0** [comment]
Thanks, just some ideas for rephrasing

**Review by ronso0 — APPROVED** [review]

**Comment by ronso0** [comment]
(just waiting for CI to be all green, anyone passing by may press merge)

**Comment by daschuer** [comment]
I have added some more entries since your review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,13 +19,39 @@
...
 * Developer Tools: Initially sort controls by group name, ascending [#12884](https://github.com/mixxxdj/mixxx/pull/12884)
-* History: Show track count and duration in sidebar [#12811](https://github.com/mixxxdj/mixxx/pull/12811)
-* Prevent removing tracks from locked playlists [#12927](https://github.com/mixxxdj/mixxx/pull/12927)
+* History: Show track count and duration in sidebar
+  [#12811](https://github.com/mixxxdj/mixxx/pull/12811)
+  [#13020](https://github.com/mixxxdj/mixxx/pull/13020)
+  [#13019](https://github.com/mixxxdj/mixxx/issues/13019)
+  [#12788](https://github.com/mixxxdj/mixxx/issues/12788)
+* Library sidebar: fix History label update [#12882](https://github.com/mixxxdj/mixxx/pull/12882) [#12880](https://github.com/mixxxdj/mixxx/issues/12880)
+* Library: allow adding new dirs while we have missing watched dir(s)
```
Comment (ronso0) [inline CHANGELOG.md:37]:
```suggestion
* Library: allow adding new directories while watched directories are missing
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,13 +19,39 @@
...
 * Remove unnecessary unpolish operation of the style, before polish the new style [#12445](https://github.com/mixxxdj/mixxx/pull/12445)
 * Developer Tools: Initially sort controls by group name, ascending [#12884](https://github.com/mixxxdj/mixxx/pull/12884)
-* History: Show track count and duration in sidebar [#12811](https://github.com/mixxxdj/mixxx/pull/12811)
-* Prevent removing tracks from locked playlists [#12927](https://github.com/mixxxdj/mixxx/pull/12927)
+* History: Show track count and duration in sidebar
+  [#12811](https://github.com/mixxxdj/mixxx/pull/12811)
+  [#13020](https://github.com/mixxxdj/mixxx/pull/13020)
+  [#13019](https://github.com/mixxxdj/mixxx/issues/13019)
+  [#12788](https://github.com/mixxxdj/mixxx/issues/12788)
+* Library sidebar: fix History label update [#12882](https://github.com/mixxxdj/mixxx/pull/12882) [#12880](https://github.com/mixxxdj/mixxx/issues/12880)
```
Comment (ronso0) [inline CHANGELOG.md:36]:
```suggestion
* Library sidebar: show track count and duration of History playlists
  [#12811](https://github.com/mixxxdj/mixxx/pull/12811)
  [#13020](https://github.com/mixxxdj/mixxx/pull/13020)
  [#13019](https://github.com/mixxxdj/mixxx/issues/13019)
  [#12788](https://github.com/mixxxdj/mixxx/issues/12788)
  [#12880](https://github.com/mixxxdj/mixxx/issues/12880)
  [#12882](https://github.com/mixxxdj/mixxx/pull/12882)
```

**Review by ronso0 — CHANGES_REQUESTED** [review]

**Review by ronso0 — APPROVED** [review]

**Comment by ronso0** [comment]
Thanks!

**Pull request merged**